workflow_scripts/test_model_zoo.py

```diff
@@ -72,14 +73,18 @@ def main():
         try:
             pull_lfs_file(model_path)
             model = onnx.load(model_path)
-            # stricter onnx.checker with onnx.shape_inference
-            onnx.checker.check_model(model, True)
-            # remove the model to save space in CIs
-            if os.path.exists(model_path):
-                os.remove(model_path)
-            # clean git lfs cache
-            run_lfs_prune()
-            print('[PASS]: {} is checked by onnx. '.format(model_name))
+            if model.ir_version < 4:
+                model = version_converter.convert_version(model, 7)
+                # stricter onnx.checker with onnx.shape_inference
+                onnx.checker.check_model(model, True)
+                # remove the model to save space in CIs
+                if os.path.exists(model_path):
+                    os.remove(model_path)
+                # clean git lfs cache
+                run_lfs_prune()
+                print('[PASS]: {} is checked by onnx. '.format(model_name))
+            else:
+                print('[SKIP]: {}.'.format(model_name))
```

Comment:
This is for testing version_converter with models from ONNX model zoo. Will remove this update before merge.